packages/f-content-cards/src/components/ContentCards.vue

```diff
@@ -539,6 +542,17 @@ export default {
             return groupIndex !== null ?
                 this.testId && `ContentCard-${this.testId}-${index}-${groupIndex}` :
                 this.testId && `ContentCard-${this.testId}-${index}`;
+        },
+
+        /**
+         * Handles logging from f-metadata (callback)
+         * @returns {function(*, *=, *=): void}
+         */
+        handleLogging (logger) {
+            // eslint-disable-next-line func-names
+            return ((type, logMessage, payload) => {
+                if (logger && logger[type]) logger[type](logMessage, null, payload);
```

Comment:
Just thinking, it might be a little easier to follow the code if you wrap the conditional code down onto a new line? It's more consistent with the style used elsewhere also.

```suggestion
                if (logger && logger[type]) {
                    logger[type](logMessage, null, payload);
                }
```

Reply:
agreed and resolved